docs/terminal/tindal.md

```diff
@@ -4,7 +4,97 @@
+Departing aircraft will be cleared to track direct to the gate, or allowed to intercept the outbound radial by 10 DME.  
+Arrivals can be cleared via the appropriate gate for their return to YPTN.
+
+| Gate Name | Ident | Bearing (°M) | Distance (nm) | Restricted Area |
+| ------ | ----------| ----------| ----------| ----------|
+| BIAK     | BIA | 320 | 30 | N/A |
+| TOP GATE     | TOP | 300 | 30 | R225D |
+| MOROTAI     | MOR | 270 | 30 | R225D |
+| TADJI     | TAD | 240 | 30 | R225D |
+| TARAKAN     | TAR | 200 | 30 | R225D |
+| BOTTOM GATE     | BOM | 190 | 30 | R225D |
+| NADZAB     | NAD | 150 | 30 | N/A |
+| MILNE     | MIL | 090 | 30 | R226A |
+| WEDGE 1     | W1 | 285 | 16 | R238 |
+| WEDGE 2     | W2 | 260 | 12 | R238 |
+| WEDGE 3     | W3 | 200 | 16 | R238 |
+
+<figure markdown>
+![Tindal Gates](img/../../aerodromes/classc/img/yptngates.png){width="700"}
+  <figcaption>Tindal Gates</figcaption>
+</figure>
+
+!!! phraseology 
+    CLAS35 has completed operations in R225, and is requesting return to YPTN via TARAKAN Gate.  
+    **TN TCU**: "CLAS35, cleared to YPTN via TARAKAN, thence TN. When ready descend to F130"   
+
+Place/Bearing Distance points in the FDR can be used for routing via the gates, with **TN** as the reference point, in the format PPPPBBBDDD.
+
+!!! example 
+    MOROTAI = TN170030  
+    NADZAB = TN150030  
+
+## Coordination
+### TRT(TRS)
+#### Airspace
+Any airspace releases from the default setup must be coordinated and agreed upon with TRT(TRS). It is also good practice to remind them of any airspace releases that may be active due to NOTAMs.
+
+#### Departures
+Voiceless for all aircraft:
+
+- Tracking via a Procedural SID terminus; and  
+- Assigned the lower of `F180` or the `RFL`
+
+!!! note
+    Aircraft are *not required* to be tracking via the **SID procedure**, simply tracking via any of the terminus waypoints (Regardless of *departure airport* or *assigned SID*) is sufficient to meet the criteria for **voiceless coordination**
+
+All other aircraft going to TRT CTA must be **Heads-up** Coordinated by TN TCU prior to the boundary.
+
+!!! phraseology
+    <span class="hotline">**TN TCU** -> **TRS**</span>: "DRGN48, request DCT BEBEX"  
+    <span class="hotline">**TRS** -> **TN TCU**</span>: "DRGN48, concur DCT BEBEX"  
+
+#### Arrivals/Overfliers
+Voiceless for all aircraft:
+
+- Tracking via **TN**; and  
+- Assigned the lower of `F130` or the `RFL`.
+
+All other aircraft coming from TRT(TRS) CTA will be **Heads-up** Coordinated to TN TCU.
+
+### TN ADC
+'Next' coordination is required from TN ADC to TN TCU for all aircraft.
+
+!!! phraseology
+    <span class="hotline">**TN ADC** -> **TN TCU**</span>: "Next, ASY01, runway 32"  
+    <span class="hotline">**TN TCU** -> **TN ADC**</span>: "ASY01, Assigned Heading Right 030, unrestricted"  
+    <span class="hotline">**TN ADC** -> **TN TCU**</span>: "Assigned Heading Heading Right 030, ASY01"  
```

Comment:
@alphadelta332 should this coord be 'heading 030 unrestricted' rather than 'assigned heading right...'?